Doc/library/functions.rst

```diff
@@ -1165,7 +1165,11 @@ are always available.  They are listed here in alphabetical order.
    *buffering* is an optional integer used to set the buffering policy.  Pass 0
    to switch buffering off (only allowed in binary mode), 1 to select line
    buffering (only usable in text mode), and an integer > 1 to indicate the size
-   in bytes of a fixed-size chunk buffer.  When no *buffering* argument is
+   in bytes of a fixed-size chunk buffer. Note that specifying a buffer size this
+   way applies for binary buffered I/O, but ``TextIOWrapper`` (i.e., files opened
+   with ``mode='r+'``) would have another buffering. For configuring text buffer
+   sizes, consider using the ``write_through`` flag for
```

Comment:
You can not change the buffer size.

"To disable buffering in the ``TextIOWrapper``, ..."

Reply:
Thanks, I dropped the 'the' since it sounds better to me without, but let me know if you'd prefer it remain

Reply:
I'm not good at English. Writing "the" correctly is very difficult to me.
So please ignore my grammar😅